buddyboss/buddyboss-platform
# Discussion reply line break issue - fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2283  .

### How to test the changes in this Pull Request:

1. Go to any forum discussion.
2. Make a reply.
3. Check the email notification received by the subscriber of that discussion
4. Notice the issue is not happening anymore.

### Proof Screenshots or Video

https://www.loom.com/share/249341f274e9465d9f00a1a20979add4

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
